src/overlays/actors/ovl_Bg_Astr_Bombwall/z_bg_astr_bombwall.h

```diff
@@ -5,15 +5,16 @@
 
 struct BgAstrBombwall;
 
-typedef void (*BgAstrBombwallActionFunc)(struct BgAstrBombwall*, GlobalContext*);
+#define OBJBgAstrBombwall_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
```

Comment:
```suggestion
#define BGASTRBOMBWALL_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
```
Don't forget to adjust the casing here and drop the `OBJ`.